lib/membrane_rtc_engine/endpoints/webrtc_endpoint.ex

```diff
@@ -180,17 +201,6 @@ if Code.ensure_loaded?(Membrane.WebRTC.EndpointBin) do
       {{:ok, spec: spec, log_metadata: opts.log_metadata}, state}
     end
 
-    @impl true
-    def handle_notification(
-          {:integrated_turn_servers, turns},
-          _from,
-          _ctx,
-          state
-        ) do
-      enforce_turns? = state.use_integrated_turn || false
-      {{:ok, notify: {:integrated_turn_servers, turns, enforce_turns?}}, state}
-    end
-
```

Comment:
Are you sure this should be deleted?

Reply:
Yup this is some unused code.  There is no notify action  `{:integrated_turn_servers, turns} in EndpointBin